target/scripts/startup/setup.d/saslauthd.sh

```diff
@@ -9,24 +9,7 @@ function _setup_saslauthd() {
   if [[ ${ACCOUNT_PROVISIONER} == 'LDAP' ]] \
   && [[ ! -f /etc/saslauthd.conf ]]; then
     _log 'trace' 'Creating /etc/saslauthd.conf'
-
-    # Create a config based on ENV
-    sed '/^.*: $/d'> /etc/saslauthd.conf << EOF
-ldap_servers: ${SASLAUTHD_LDAP_SERVER:=${LDAP_SERVER_HOST}}
-ldap_auth_method: ${SASLAUTHD_LDAP_AUTH_METHOD:=bind}
-ldap_bind_dn: ${SASLAUTHD_LDAP_BIND_DN:=${LDAP_BIND_DN}}
-ldap_bind_pw: ${SASLAUTHD_LDAP_PASSWORD:=${LDAP_BIND_PW}}
-ldap_search_base: ${SASLAUTHD_LDAP_SEARCH_BASE:=${LDAP_SEARCH_BASE}}
-ldap_filter: ${SASLAUTHD_LDAP_FILTER:=(&(uniqueIdentifier=%u)(mailEnabled=TRUE))}
-ldap_start_tls: ${SASLAUTHD_LDAP_START_TLS:=no}
-ldap_tls_check_peer: ${SASLAUTHD_LDAP_TLS_CHECK_PEER:=no}
-ldap_tls_cacert_file: ${SASLAUTHD_LDAP_TLS_CACERT_FILE}
-ldap_tls_cacert_dir: ${SASLAUTHD_LDAP_TLS_CACERT_DIR}
-ldap_password_attr: ${SASLAUTHD_LDAP_PASSWORD_ATTR}
-ldap_mech: ${SASLAUTHD_LDAP_MECH}
-ldap_referrals: yes
-log_level: 10
-EOF
+    _create_config_saslauthd
```

Comment:
Note for self, the ENV condition above and comment above it may need to be adjusted with this PR, as mentioned [here](https://github.com/docker-mailserver/docker-mailserver/pull/3522/files#r1312480257).

> Technically instead of `ACCOUNT_PROVISIONER=LDAP`, this should be for `SASLAUTHD_MECHANISMS=ldap`, as the two features serve a separate purpose? 🤷‍♂️ (_although I'm not sure when you'd only have Postfix auth against LDAP while using a different `ACCOUNT_PROVISIONER`_).